winforms/tests_backend/widgets/base.py

```diff
@@ -56,10 +55,19 @@ def color(self):
 
     @property
     def background_color(self):
-        if self.native.BackColor == Color.Transparent:
-            return TRANSPARENT
-        else:
-            return toga_color(self.native.BackColor)
+        return (
+            toga_color(self.native.BackColor),
+            toga_color(self.impl.interface.parent._impl.native.BackColor),
```

Comment:
`self.impl.interface` can be simplified to `self.widget`.

Reply:
Thanks!